synthtool/languages/java.py

```diff
@@ -23,12 +23,11 @@
 from synthtool.gcp import common, partials, pregenerated, samples, snippets
 from synthtool.log import logger
 from pathlib import Path
-from typing import Any, Optional, Dict, List
+from typing import Any, Optional, Dict, Iterable, List
 
 JAR_DOWNLOAD_URL = "https://github.com/google/google-java-format/releases/download/google-java-format-{version}/google-java-format-{version}-all-deps.jar"
 DEFAULT_FORMAT_VERSION = "1.7"
-GOOD_LICENSE = """
-/*
+GOOD_LICENSE = """/*
  * Copyright 2020 Google LLC
```

Comment:
Will we have to update "2020" at some future date?

Reply:
Yes, but wanted to keep it separate from this bugfix

Reply:
Ok.